src/config.c

```diff
@@ -425,9 +425,10 @@ CONFIG_SETTER(setGcPolicy) {
   if (!strcasecmp(policy, "DEFAULT") || !strcasecmp(policy, "FORK")) {
     config->gcPolicy = GCPolicy_Fork;
   } else if (!strcasecmp(policy, "LEGACY")) {
-    config->gcPolicy = GCPolicy_Sync;
+    QueryError_SetError(status, QUERY_EPARSEARGS, "Legacy GC policy is no longer supported (since 2.6.0)");
```

Comment:
Is this a breaking change, or is it already deprecated?
Seems we also need to update the documentation
https://redis.io/docs/stack/search/configuring/#GC_POLICY

Reply:
Done